css/style.css

```diff
@@ -1,10 +1,12 @@
 .fixed-top.scrolled {
     background-color: #222324 !important;
-    transition: background-color 500ms linear;
 }
 .text-white.dark {
     color: #222324 !important;
 }
+.fixed-top {
+    transition: background-color 500ms linear;
```

Comment:
Why move it to a separate class?

Reply:
so that it shows transition on close of menu as well.